aikau/src/main/resources/alfresco/documentlibrary/AlfBreadcrumbTrail.js

```diff
@@ -268,21 +269,22 @@ define(["dojo/_base/declare",
                this.alfSubscribe(this.metadataChangeTopic, lang.hitch(this, this.onCurrentNodeChange));
             }
 
+            if (this.currentPath)
+            {
+               this.renderPathBreadcrumbTrail();
+            }
```

Comment:
This block not new, just moved above the hash handlers to ensure default state loaded before hash applied.